en/releases/centreon-core.md

```diff
@@ -19,6 +19,19 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 20.10.14
+
+#### Bug fixes
+
+- [Authentication] Fixed PHP error when debug is enabled with OIDC authentication
+- [CLAPI] Fixed LDAP configuration ID for export and import
+- [Configuration] Fixed Centreon Broker configuration that could be empty or deleted
+- [Configuration] Fixed Centreon Broker configuration that could be empty or deleted
+- [Core] REmove PHP warning in logs
+- [Resources Status] Fixed access to action URLs
+- [Resources Status] Fixed access to actions buttons
+- [UX] Non admin user do not have the same submenu subsections
+
```

Comment:
Maybe MON-11139 : [Dependency to upgrade on : centreon/centreon] - composer prod. bump nelmio/cors-bundle from 1.5.6 to 2.1.1 should appear

Reply:
no this is technical